Lotto/Lotto/Calculator/CalculatorError.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CalculatorError.swift
+//  Lotto
+//
+//  Created by 카카오VX on 2022/04/29.
+//
+
+import Foundation
+
+enum CalculatorError: Error {
+    case unSupportedOperand
+    case unSupportedNegativeNumber
+    case unSupportedSeparator
+    
+    var message: String {
+        switch self {
+        case .unSupportedOperand:
+            return "숫자 이외의 값은 입력할 수 없습니다."
```

Comment:
```suggestion
            return "지원하지 않는 피연산자입니다.(숫자만 입력가능)."
```
은 어떠신가요..?! 에러케이스가 unSupported로 시작되어서 의견드려봅니다 😊